GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -633,16 +664,6 @@ private void saveAsToolStripMenuItem1_Click(object sender, EventArgs e)
             }
         }
 
-
-        private bool DeleteSelectedDiffFiles()
```

Comment:
OK to remove, just a comment: This was purposely not removed as I want to have Keyboard shortcut here. A refrence from FormBrowse is needed

Reply:
this PR is concentrating on moving logic out of the UI.
a following PR can concentrate on adding keyboard shortcuts